_inc/client/plans/single-product-backup.jsx

```diff
@@ -12,25 +11,37 @@ import { get } from 'lodash';
  */
 import Button from 'components/button';
 import ExternalLink from 'components/external-link';
+import PlanPrice from 'components/plans/plan-price';
 
 import './single-product-backup.scss';
 
 export function SingleProductBackup( { products, upgradeLinks } ) {
 	const [ selectedBackupType, setSelectedBackupType ] = useState( 'real-time' );
 
-	const backupPlanPrices = {
-		jetpack_backup_daily: {
-			monthly: get( products, [ 'jetpack_backup_daily_monthly', 'cost' ], '' ),
-			yearly: get( products, [ 'jetpack_backup_daily', 'cost' ], '' ),
+	const billingTimeFrame = __( 'per year' );
+	const currencyCode = get( products, [ 'jetpack_backup_daily', 'currency_code' ], '' );
+
+	const priceDaily = get( products, [ 'jetpack_backup_daily', 'cost' ], '' );
+	const priceDailyMonthly = get( products, [ 'jetpack_backup_daily_monthly', 'cost' ], '' );
+	const priceDailyMonthlyPerYear = priceDailyMonthly * 12;
```

Comment:
In the case where priceDaily doesn't exist it becomes `''`, so `priceDailyMonthly` should probably also be `''` in this case instead of `0`.

```suggestion
	const priceDailyMonthlyPerYear = '' === priceDailyMonthly ? '' : priceDailyMonthly * 12;
```